main/solution/backend/config/infra/functions.yml

```diff
@@ -156,6 +156,7 @@ apiHandler:
     APP_ENV_TYPE_CONFIGS_BUCKET_NAME: ${self:custom.settings.envTypeConfigsBucketName}
     APP_ENV_BOOTSTRAP_BUCKET_NAME: ${self:custom.settings.environmentsBootstrapBucketName}
     APP_CUSTOM_USER_AGENT: ${self:custom.settings.customUserAgent}
+    APP_ROOT_USER_NAME: ${self:custom.settings.rootUserName}
```

Comment:
Where do we use `APP_ROOT_USER_NAME `?

Reply:
We use it [here](https://github.com/awslabs/service-workbench-on-aws/blob/27523aee8b40670d169154d75bcc681aa079d29d/addons/addon-base/packages/services/lib/db-password/db-password-service.js#L50) to grab the root username. We also use it on line 62 in the same file.

Reply:
Got it, I keep forgetting that the convention changes to camel case without APP

Reply:
Hi all, I happened to see this and wanted to give you a little bit of context around why we have APP_
These are unix environment variables that are passed to the lambda node.js unix process.  We add APP_ so that our environment variables don't collide with any ones provided by lambda.